src/language/parser.js

```diff
@@ -206,6 +206,11 @@ class Parser {
       ),
       loc: this.loc(start),
     };
+    const comments = this._lexer.commentsList;
+    if (comments.length >= 1) {
```

Comment:
This check is not needed just return an empty array.
Maybe it's not ideal but this is how all other arrays inside AST are handled.

Reply:
Yes, but then we would have to modify those tests having where comments is an empty array, right? such as `buildASTSchema-test.js`

Reply:
@nveenjain No, please look at other optional arrays in AST.
We return empty arrays in parser but allow users to create nodes where such arrays can be omitted.